xlml/utils/xpk.py

```diff
@@ -142,14 +141,16 @@ def wait_for_workload_completion(
       elif pod.status.phase in ["Unknown"]:
         raise RuntimeError(f"Bad pod phase: {pod.status.phase}")
   finally:
-    # Print the logs of the last pod checked - either the first failed pod or
-    # the last successful one.
-    logs = core_api.read_namespaced_pod_log(
-        name=pod.metadata.name, namespace=pod.metadata.namespace
-    )
-    logging.info(f"Logs for pod {pod.metadata.name}:")
-    for line in logs.split("\n"):
-      logging.info(line)
+    # TODO(jonbolin): log printing for GPUs, which have multiple containers
```

Comment:
logs from 1 pod works fine right? I don't have source to the local run.

Reply:
Ah yeah, single-container jobs work: https://screenshot.googleplex.com/5JcEKeSeBBSohho